src/ui/public/indices/get_indices.js

```diff
@@ -1,6 +1,10 @@
 import { pluck, reduce, size } from 'lodash';
 
-const getIndicesFromResponse = json => {
+const getIndexNamesFromAliasesResponse = json => {
+  if (json.status === 404) {
```

Comment:
Sure, but can you define the right under `export function IndicesGetIndicesProvider(esAdmin) {`?